src/main/client/src/dashBoard/DashBoard.js

```diff
@@ -104,6 +105,9 @@ class DashBoard extends Component {
             />
           </div>
         )}
+        <Button onClick={() => axios.post("/api/borrow/collect/6")}>
+          Funky Button
+        </Button>
```

Comment:
Was this just for testing?

Reply:
we definitely tried not to commit that ...

Reply:
🤷